ui/src/views/sql-query-editor/components/sql-editor-section.tsx

```diff
@@ -80,6 +81,10 @@ const SQLEditorSection: React.FC<SQLEditorSectionProps> = ({ onEnterKey }: SQLEd
             }}
           />
         </div>
+
+        <div className='border-l' style={{ minWidth: '340px' }}>
```

Comment:
```suggestion
        <div className='border-l w-full' style={{ minWidth: '340px' }}>
```

I _think_ this should be enough to make it fill the space?

Reply:
I solved this here [f72ceaf](https://github.com/mergestat/mergestat/pull/832/commits/f72ceaf560a6b7ed2e04b88a1795562c90444444) (Adding `flex-1`)